src/plugins/dialog/math.js

```diff
@@ -0,0 +1,280 @@
+        context.math._mathExp = null;
+        math_button.addEventListener('mousedown', function (e) { e.stopPropagation(); }, false);
+
+        /** add event listeners */
+        math_dialog.querySelector('.se-btn-primary').addEventListener('click', this.submit.bind(core));
+        math_button.addEventListener('click', this.onClick_mathBtn.bind(core));
+
+        /** append html */
+        context.dialog.modal.appendChild(math_dialog);
+        context.element.relative.appendChild(math_button);
+
+        /** empty memory */
+        math_dialog = null, math_button = null;
+    },
+
+    /** dialog */
+    setDialog: function () {
+        const lang = this.lang;
+        const dialog = this.util.createElement('DIV');
+
+        dialog.className = 'se-dialog-content';
+        dialog.style.display = 'none';
+        dialog.innerHTML = `
+        <form class='editor_math'>
+            <div class='se-dialog-header'>
+                <button type='button' data-command='close' class='close' aria-label='Close' title='${lang.dialogBox.close}'>
+                    <i aria-hidden='true' data-command='close' class='se-icon-cancel'></i>
+                </button>
+                <span class='se-modal-title'>${lang.dialogBox.mathBox.title}</span>
+            </div>
+            <div class='se-dialog-body'>
+                <div class='se-dialog-form'>
+                    <label>${lang.dialogBox.mathBox.inputLabel} (<a href='https://katex.org/docs/supported.html' target='_blank'>KaTeX</a>):</label>
+                    <textarea style='height: 4rem;border: 1px solid #ccc !important;' class='se-input-form _se_math_exp' type='text'></textarea>
+                </div>
+                <div class='se-dialog-form'>
+                    <label>${lang.dialogBox.mathBox.fontSizeLabel}:</label>
+                    <select style='width:6em;margin-left: 1em;' class='_se_math_size'>
+                        <option value='1em'>1</option>
+                        <option value='1.5em'>1.5</option>
+                        <option value='2em'>2</option>
+                        <option value='2.5em'>2.5</option>
+                    </select>
+                </div>
+                <div class='se-dialog-form'>
+                    <label>${lang.dialogBox.mathBox.previewLabel}:</label>
+                    <p style='font-size:13px;' class='_se_math_preview'></p>
+                </div>
+            </div>
+            <div class='se-dialog-footer'>
+                <button type='submit' class='se-btn-primary' title='${lang.dialogBox.submitButton}'><span>${lang.dialogBox.submitButton}</span></button>
+            </div>
+        </form>
+        `;
+
+        return dialog;
+    },
+
+    /** modify controller button */
+    setController_MathButton: function () {
+        const lang = this.lang;
+        const math_btn = this.util.createElement('DIV');
+
+        math_btn.className = 'se-controller se-controller-link';
+        math_btn.innerHTML = `
```

Comment:
\` << is not supported in IE11.
It's cumbersome, but it should be in the same format this code [dialog/link.js#L59](https://github.com/JiHong88/SunEditor/blob/master/src/plugins/dialog/link.js#L59)

Reply:
This is very annoying.
I can do it, so you don't have to do it :)

Reply:
done